src/main/java/com/neroreflex/plugins/HackerNews.java

```diff
@@ -76,22 +75,29 @@ private synchronized void sendHackerNews() {
     }
 
     protected final void onCall(String user, String channel, Vector<String> args) {
-        this.sendHackerNews();
+        this.sendHackerNews(channel);
     }
 
-    protected final void onInitalize() {
+    protected final void onInitialize() {
         Calendar cal = Calendar.getInstance(TimeZone.getTimeZone("Europe/Rome"));
-        cal.set(Calendar.HOUR_OF_DAY, 8); // start at 8 AM
+        int currHour = cal.get(Calendar.HOUR_OF_DAY);
+        //le notifiche vengono inviate alle 2, 8, 14, 20; la formula determina il prossimo orario più vicino tra quelli.
+        int startHour = ((int)Math.floor(((currHour - 2)/6.0)) + 1)* 6 + 2;
```

Comment:
Giusto avevo dimenticato, in tal caso non serve il modulo ma il commento prima rimane valido